packages/auth/src/grant/routes.ts

```diff
@@ -384,12 +411,32 @@ async function continueGrant(
   const access = await accessService.getByGrant(grant.id)
 
   // TODO: add "continue" to response if additional grant request steps are added
-  ctx.body = {
+  ctx.body = createGrantBody(
+    config.authServerDomain,
+    grant,
+    access,
+    accessToken
+  )
+}
+
+function createGrantBody(
+  domain: string,
+  grant: Grant,
+  access: Access[],
+  accessToken: AccessToken
+) {
+  return {
     access_token: {
       value: accessToken.value,
-      manage: config.authServerDomain + `/token/${accessToken.managementId}`,
+      manage: domain + `/token/${accessToken.managementId}`,
       access: access.map((a: Access) => accessToBody(a)),
-      expiresIn: accessToken.expiresIn
+      expiresIn: accessToken.expiresIn,
+      continue: {
```

Comment:
We can exclude the `continue` object if there's no interaction required and the AS can just supply an `access_token` instead. Also vice/versa for if interaction _is_ required. It's basically an either/or situation between `continue` and `access_token` with how we're using GNAP.

Reply:
See above.